src/core/connection.c

```diff
@@ -5715,6 +5718,9 @@ QuicConnParamSet(
                 QuicLibraryGetBinding(
 #ifdef QUIC_COMPARTMENT_ID
                     Connection->Configuration->CompartmentId,
+#endif
+#ifdef QUIC_OWNING_PROCESS
+                    Connection->Configuration->OwningProcess,
```

Comment:
Should this always be `NULL`? In this case, the connection is already started, which means its attached to a binding which already has a socket that is created.

Reply:
Since a new socket could be opened (if the connection is going to use a new local address) this will remain.